test/project-service.ts

```diff
@@ -147,12 +145,39 @@ describe("Project Service Tests", () => {
 			let options = projectIntegrationTest.testInjector.resolve("options");
 
 			options.path = tempFolder;
-			options.copyFrom = projectIntegrationTest.getDefaultTemplatePath().wait();
+			options.copyFrom = projectIntegrationTest.getDefaultTemplatePath("tns-template-hello-world").wait();
 			options.appid = "my.special.id";
 
 			projectIntegrationTest.createProject(projectName).wait();
 			projectIntegrationTest.assertProject(tempFolder, projectName, options.appid).wait();
 		});
+		it("creates ios project and tests post-install sandboxing of CocoaPods setup", () => {
+			if (require("os").platform() !== "darwin") {
+				console.log("Skipping CocoaPods sandbox test. It works only on darwin.");
+				return;
+			}
+
+			let testInjector = createTestInjector();
+			let fs: IFileSystem = testInjector.resolve("fs");
+			let config = testInjector.resolve("config");
+			let childProcess = testInjector.resolve("childProcess");
+			let projectIntegrationTest = new ProjectIntegrationTest();
+			let workingFolderPath = temp.mkdirSync("ios_project");
+
+			let iosTemplatePath = path.join(projectIntegrationTest.getDefaultTemplatePath("tns-ios").wait(), "framework/");
+			childProcess.exec(`cp -R ${iosTemplatePath} ${workingFolderPath}`, { cwd: workingFolderPath }).wait();
+			fs.writeFile("/tmp/Podfile/testFile.txt", "Test content.").wait();
+
+			let postInstallCommmand = `\`cat /tmp/Podfile/testFile.txt > ${workingFolderPath}/copyTestFile.txt && rm -rf /tmp/Podfile\``;
+			let podfileContent = `post_install do |installer_representation| ${postInstallCommmand} end`;
+			fs.writeFile(path.join(workingFolderPath, "Podfile"), podfileContent).wait();
+
+			let podTool = config.USE_POD_SANDBOX ? "sandbox-pod" : "pod";
+			childProcess.spawnFromEvent(podTool,  ["install"], "close", { cwd: workingFolderPath, stdio: 'inherit' }).wait();
```

Comment:
Shouldn't we call here `executePodInstall` function from `iosProjectService`? This way if we break something the test will fail.